stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/factory/ChannelListViewModelFactory.kt

```diff
@@ -17,8 +17,8 @@ import io.getstream.chat.android.ui.channel.list.viewmodel.ChannelListViewModel
  * @param filter How to filter the channels.
  * @param sort How to sort the channels, defaults to last_updated.
  * @param limit How many channels to return.
- * @param messageLimit The number of messages to fetch for each channel.
- * @param chatEventHandler The instance of [ChatEventHandler] that will be used to handle channel updates event for this combination of [sort] and [filter].
+ * @param memberLimit The number of members per channel.
```

Comment:
* @param messageLimit The number of messages to fetch for each channel.

Reply:
nice catch!